pmm-app/src/shared/style.less

```diff
@@ -31,8 +31,8 @@
 @table-selected-row-bg: gray;
 @border-color-split: rgb(40, 40, 40);
 @table-row-hover-bg: transparent;
-@select-item-selected-bg: grey !important;
-@select-item-active-bg: grey !important;
+@select-item-selected-bg: #2D2E2F !important;
```

Comment:
Note: since this is global, let's verify how other antd select components render (on other panels, elsewhere)

Reply:
Correct me if I'm wrong but I only see one other use of antd Select and it's on the same page.